Gemfile.lock

```diff
@@ -310,7 +310,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 3.1.0p0
+   ruby 3.1.1p18
 
 BUNDLED WITH
    2.3.7
```

Comment:
You can also start using the latest bundler: https://rubygems.org/gems/bundler/.

I think this should do the job:

```
gem install bundler
BUNDLER_VERSION=2.3.11 bundle install
```

Reply:
Forgot, changed :) 

Yes, bundler also will be upgraded